### PR TITLE
Add 01Space board for Circuitpython

### PR DESCRIPTION
**Pull request by Neradoc**
This is a PID request for running Circuitpython on the 01Space RP2040 0.42 LCD board.
https://github.com/01Space/RP2040-0.42LCD

The board is not open hardware, the maker has no VID and is not requesting a PID from Rapsberry Pi.

### CODE REVIEW

File: 1209/42B0/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: pid
+title: RP2040 0.42 LCD
+owner: 01space
```
Comment (tannewt) [inline 1209/42B0/index.md:4]: Please have yourself or "CircuitPython" as the owner so that it is clear that this PID wasn't granted to the manufacturer.

**Review by tannewt — CHANGES_REQUESTED** [review]
Please use a different org.

File: 1209/42B0/index.md (hunk 1 above)
Comment (Neradoc) [reply 1209/42B0/index.md:4]: Ah, so should I not create the org then either ?

Comment (tannewt) [reply 1209/42B0/index.md:4]: Make one for yourself or CP but not the manufacturer.

**Comment by Neradoc** [comment]
OK I'll think about it.

**Pull request closed without merging**